### DISCUSSION TITLE
Animation stops after a while even after setting loop to true

### DISCUSSION THREAD

**Discussion by randyelorm** [nandorojo/moti#213] [Q&A]
I use Moti with react native expo and I've noticed that after setting loop = true, the animation still stops after a while. I'd appreciate help with this. Thanks so much.

**Comment by nandorojo** [comment]
is loop `true` on the initial render, or does it change over time?

**Comment by randyelorm** [comment]
loop is true on the initial render.. it does not change over time.
    On Monday, 18 July 2022 at 01:26:31 GMT, Fernando Rojo ***@***.***> wrote:  
 
 


is loop true on the initial render, or does it change over time?

**Reply by nandorojo** [reply]
can you show the code / make a snack that reproduces it? what if you set the easing to linear?
